code/game/objects/items/powerfist.dm

```diff
@@ -19,6 +19,7 @@
 	var/gasperfist = 3
 	var/obj/item/tank/internals/tank = null //Tank used for the gauntlet's piston-ram.
 	var/baseforce = 20
+	var/lock //if this is locked to the user
```

Comment:
This isn't documented in the changelog.
It's actually a major buff to the power fist as well.

Have a define and undefine stating that it starts unlocked.

Reply:
i would say its a double edged sword but will update the pr as this instantly calls you out as an antag when put on as there is no removal.